modules/users/client/components/utils/networks.js

```diff
@@ -0,0 +1,35 @@
+/**
+  * Determine if given user handle for Warmshowers is an id or username
+  * @link https://github.com/Trustroots/trustroots/issues/308
+  */
+function isWarmshowersId(profile) {
+  // WarmShowers id should contain only digits
+  return /^[0-9]*$/.test(profile.extSitesWS);
+}
+
+/**
+     * Check if there are additional accounts
```

Comment:
I wonder if we can have eslint rule for this. I wouldn't expect this indentation (here and line 18).

This is truly a [nitpicking](https://en.wikipedia.org/wiki/Nitpicking). I don't feel a need to have this changed. :)

Reply:
Should we add: 'multiline-comment-style': ["error", "starred-block"] ? :slightly_smiling_face:

Reply:
There is a rule for that? Great. I'd say let's add it, but not necessarily now.